public/app/features/alerting/unified/state/actions.ts

```diff
@@ -456,6 +459,32 @@ export const saveRuleFormAction = createAsyncThunk(
             const rulerConfig = getDataSourceRulerConfig(thunkAPI.getState, GRAFANA_RULES_SOURCE_NAME);
             const rulerClient = getRulerClient(rulerConfig);
             identifier = await rulerClient.saveGrafanaRule(values, evaluateEvery, existing);
+
+            // track if the user switched from simplified routing to policies routing or vice versa
+            if (isGrafanaRulerRule(existing?.rule)) {
+              const ga = existing?.rule.grafana_alert;
+              const existingWasUsingSimplifiedRouting = Boolean(ga?.notification_settings?.receiver);
+              const newValuesUsesSimplifiedRouting = values.manualRouting;
+              const shouldTrackSwitchToSimplifiedRouting =
+                !existingWasUsingSimplifiedRouting && newValuesUsesSimplifiedRouting;
+              const shoouldTrackSwitchToPoliciesRouting =
+                existingWasUsingSimplifiedRouting && !newValuesUsesSimplifiedRouting;
+
+              if (shouldTrackSwitchToSimplifiedRouting) {
+                trackSwitchToSimplifiedRouting({
```

Comment:
nit: can we move `grafana_version`, `org_id` and `user_id` to the `Analytics.ts` instead? Those seem like global values.